### PR TITLE
Update UI: Preference and Help

### PR DESCRIPTION
**Pull request by SilviaLi1225**
Update UI: Preference and Help

### CODE REVIEW

File: TobiiTest/Preference.xaml [hunk 1]
```diff
@@ -32,15 +32,28 @@
...
+            <ComboBox.Background>
+                <LinearGradientBrush EndPoint="0,1" StartPoint="0,0">
+                    <GradientStop Color="#FFF0F0F0" Offset="0"/>
+                    <GradientStop Color="#FFFBF8F8" Offset="1"/>
+                </LinearGradientBrush>
+            </ComboBox.Background>
+        </ComboBox>
+
+        <Label Name="two" Content="Magnification factor" HorizontalAlignment="Left" Margin="25,186,0,0" VerticalAlignment="Top"/>
+        <TextBox x:Name="magnify" HorizontalAlignment="Left" Margin="194,186,0,0" TextWrapping="Wrap" Text="" VerticalAlignment="Top" Width="162" Height="25" FontSize="10"/>
```
Comment (Nerator) [inline TobiiTest/Preference.xaml]: Can you add `NumberValidationTextBox` handler to this like in other two textboxes (Lines 12 and 13)? This is to ensure, that only numbers are able to be typed in these textboxes. I didn't check it myself though.
```suggestion
        <TextBox x:Name="magnify" HorizontalAlignment="Left" Margin="194,186,0,0" TextWrapping="Wrap" Text="" VerticalAlignment="Top" Width="162" Height="25" FontSize="10"  PreviewTextInput="NumberValidationTextBox"/>
```

File: TobiiTest/Preference.xaml.cs [hunk 2]
```diff
@@ -53,6 +53,11 @@ public Preference(Preferences pr)
             ssSizeCB.SelectionChanged += SsSizeCB_SelectionChanged;
             sizeXTB.TextChanged += SizeXTB_TextChanged;
             sizeYTB.TextChanged += SizeYTB_TextChanged;
+
+            setKey.SelectedItem = setKey.Items.IndexOf(prefs.Get("key"));
```
Comment (Nerator) [inline TobiiTest/Preference.xaml.cs]: `SelectedItem` is an object in the ComboBox. And you try to assign number (`IndexOf`) to it.
```suggestion
            setKey.SelectedIndex = setKey.Items.IndexOf(prefs.Get("key"));
```

File: TobiiTest/Preference.xaml.cs [hunk 3]
```diff
@@ -65,6 +70,10 @@ private void InitComboBoxes()
             translatorCB.Items.Clear();
             translatorCB.Items.Add("Google");
             translatorCB.Items.Add("Microsoft");
+            setKey.Items.Clear();
+            setKey.Items.Add("Left Ctrl");
+            setKey.Items.Add("Left Shift");
+            setKey.Items.Add("Left Ctrl+Alt");
```
Comment (Nerator) [inline TobiiTest/Preference.xaml.cs]: Ctrl+Alt will be hard to detect. Better stick with just one key, not key combinations. You can add right ctrl and shift, for example.

File: TobiiTest/Preference.xaml (hunk 1 above)
Comment (SilviaLi1225) [reply TobiiTest/Preference.xaml]: OK.

File: TobiiTest/Preference.xaml.cs (hunk 3 above)
Comment (SilviaLi1225) [reply TobiiTest/Preference.xaml.cs]: I have changed to "Right Shift" and "Right Ctrl"

**Pull request merged**